src/components/Dashboard/dashboardLayout.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import Dboard from '../../components/Layouts/dashboard';
+
+export default function Layout(props) {
```

Comment:
This is a normal component that uses a layout.
Change the naming to something else like ex: TripsPage.
Change both the file naming and component naming